backend/media/src/main/java/co/airy/core/media/Resolver.java

```diff
@@ -0,0 +1,46 @@
+package co.airy.core.media;
+
+import co.airy.kafka.streams.KafkaStreamsWrapper;
+import co.airy.log.AiryLoggerFactory;
+import org.apache.kafka.streams.KafkaStreams;
+import org.apache.kafka.streams.StreamsBuilder;
+import org.slf4j.Logger;
+import org.springframework.beans.factory.DisposableBean;
+import org.springframework.boot.context.event.ApplicationStartedEvent;
+import org.springframework.context.ApplicationListener;
+import org.springframework.stereotype.Component;
+
+@Component
+public class Resolver implements ApplicationListener<ApplicationStartedEvent>, DisposableBean {
+    private final Logger log = AiryLoggerFactory.getLogger(Resolver.class);
+
+    private static final String appId = "media.Resolver";
+    private final KafkaStreamsWrapper streams;
+
+    public Resolver(KafkaStreamsWrapper streams) {
+        this.streams = streams;
+    }
+
+    private void startStream() {
+        final StreamsBuilder builder = new StreamsBuilder();
+
+        streams.start(builder.build(), appId);
+    }
```

Comment:
We've been skipping this method in other streaming apps (and directly using the onApplicationEvent), I would probably do this year as well